modules/libs/entities/school.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne, JoinColumn } from 'typeorm';
+import { Organization } from './organization'; // Assuming you have an Organization entity
+
+@Entity({ name: 'schools' })
+export class School {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
```

Comment:
I would make it not nullable because the school MUST have a name.

```suggestion
  @Column({
      nullable: false,
  })
```